packages/postgres-database/src/simcore_postgres_database/migration/versions/53e095260441_add_projects_access_rights_and_.py

```diff
@@ -21,7 +22,7 @@ def upgrade():
         "projects",
         sa.Column(
             "access_rights",
-            sa.dialects.postgresql.JSONB(),
+            JSONB,
```

Comment:
why is this an issue? beside the fact that it was JSONB() vs JSON? and what are the implications on the migrations that are already done?

Reply:
AFAIK there are no implications. 
JSONB() vs JSON is for sqlalchemy  ``Column`` equivalent. 
Note that we do the same with other types